packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -155,6 +155,13 @@ export class InProgressComponent extends React.Component<
     window.history.pushState(null, document.title, window.location.href)
   }
 
+  componentWillMount() {
+    window.history.pushState(null, '', window.location.href)
+    window.onpopstate = function () {
+      window.history.go(1)
+    }
+  }
+
```

Comment:
Why do we need this? Could you explain this a bit?

Reply:
a little function to replace history to the present window location so if you click on the browser's back button it keeps you on the same page..also observed that the popStateListener() function handles that already..so the code is redundant and removed.